integration-tests/examples/v4l2.rs

```diff
@@ -0,0 +1,77 @@
+use anyhow::Result;
+use serde_json::json;
+use smelter_api::Resolution;
+
+use integration_tests::{
+    examples::{self, run_example},
+    ffmpeg::start_ffmpeg_receive_h264,
+};
+
+const VIDEO_RESOLUTION: Resolution = Resolution {
+    width: 1920,
+    height: 1080,
+};
+
+const OUTPUT_PORT: u16 = 8002;
+
+fn main() {
+    run_example(client_code);
+}
+
+fn client_code() -> Result<()> {
+    start_ffmpeg_receive_h264(Some(OUTPUT_PORT), None)?;
+
+    examples::post(
+        "input/input_1/register",
+        &json!({
+            "type": "v4l2",
+            "format": "yuyv",
+            "resolution": {
+                "width": VIDEO_RESOLUTION.width,
+                "height": VIDEO_RESOLUTION.height,
+            },
+            "path": "/dev/video0",
+            "framerate": 30,
+        }),
+    )?;
+
+    examples::post(
+        "output/output_1/register",
```

Comment:
switch back to rtmp or print url to whep that would be able to open whep.html with appropriate params